rllib/agents/ars/ars.py

```diff
@@ -228,10 +228,17 @@ def validate_config(self, config: TrainerConfigDict) -> None:
                 "`NoFilter` for ARS!")
 
     @override(Trainer)
-    def _init(self, config, env_creator):
+    def setup(self, config):
+        # Validate our config dict.
         self.validate_config(config)
+
+        # Generate `self.env_creator` callable to create an env instance.
+        self._get_env_creator_from_env_id(self._env_id)
```

Comment:
for whatever reason this function isn't available on master, but I also didn't find its definition in this pr diff, but only when checking out this branch. Strange :(

Reply:
Could be a mistake by me when splitting my local branch (which contained more changes).

Reply:
fixed.